resources/faq.md

```diff
@@ -0,0 +1,162 @@
+
+How do I know the primary key of my index? [Check this route](/references/indexes.md#get-one-index). The `null` value means it has not been defined yet.
+
+By default, the primary key will be inferred in the first document sent. MeiliSearch will search for an attribute that contains the string `id` in a case-insensitive manner (e.g., `uid`, `MovieId`, `ID`, `123id123`). If none has been found, no documents will be added.
+
+If you get a `document id is missing` error, the primary key was not recognized. This means your primary key is wrongly formatted. Sending primary key's name as a query parameter [when adding documents](/references/documents.md#add-or-replace-documents) should solve this issue.
+
+Note that the primary key value must contain only `A-Z a-z 0-9` and `-_` characters.
+
+Wrong:
+```json
+"id": "@BI+* ^5h2%"
+```
+
+Good:
+```json
+"id": "_Aabc012_"
+```
+
+See more elaborate [information about the primary key](/guides/main_concepts/documents.md#primary-key).
+
+## I have uploaded my documents, but I get no result when I search in my index.
+
+Your documents upload probably failed.<br>
+To understand what happened, please check this [answer](/resources/faq.md#i-did-a-call-to-an-api-route-and-i-only-got-an-updateid-as-a-response-what-does-it-mean).
+
+## Does MeiliSearch deliver an interface to search in my documents?
+
+Yes, a web interface is available on the default address and port of your MeiliSearch instance.
+
+All you need to do is open your web browser and enter MeiliSearch’s address to visit it locally. This will lead you to a web page with a search bar that will allow you to search in a selected index.
+
+Since the production environment requires an API-key for searching, the web interface is only available in development mode.
+
+Here is more information about the [MeiliSearch web interface](/guides/advanced_guides/web_interface.md).
+
+## I do not understand the relevancy of my search results.
+
+The search responses are sorted according to a set of consecutive rules called ranking rules.<br>
+Here is more information about the [relevancy of MeiliSearch](/guides/main_concepts/relevancy.md).
+
+MeiliSearch applies these ranking rules in default order. This order can be modified. Furthermore, these rules can be deleted and new ones can be added.<br>
+All of the ranking rules can be modified via the [dedicated routes in the MeiliSearch API](/references/ranking_rules.md).
+
+## Do you provide a public roadmap for MeiliSearch and its integration tools?
+
+Yes, as MeiliSearch and its integration tools are opensource, we maintain a [public roadmap](https://github.com/orgs/meilisearch/projects/2) for the general features we plan.
+
+For more accurate features and issues, everything is detailed in the issues of all our GitHub repositories.
+
+## How can I contact the MeiliSearch team?
+
+There are many ways to contact us:
+- At [bonjour@meilisearch.com](mailto:bonjour@meilisearch.com): English or French are welcome! 🇫🇷 🇬🇧
```

Comment:
I'm not sure I'm trying to find the grammar rule. 
```suggestion
- At [bonjour@meilisearch.com](mailto:bonjour@meilisearch.com): English or French is welcome! 🇫🇷 🇬🇧
```

Reply:
Grammarly says nothing